Program.cs

```diff
@@ -1,15 +1,23 @@
-using System.IO;
-﻿using System;
-using System.Net.Sockets;
+using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 using UdtSharp;
 
 namespace p2pcopy
 {
     class Program
     {
-        static void Main(string[] args)
+        private static int _defaultTaskCount = 10;
+        private static readonly List<Socket> Sockets = new List<Socket>();

```

Comment:
nit: I believe the current coding style of p2pcopy uses camelCase (not PascalCase) for fields

Reply:
Well, I follow the Resharper naming convention, for static readonly fields it suggests PC, I'll change to cC.